### DISCUSSION TITLE
Lettura risposte di un questionario

### DISCUSSION THREAD

**Discussion by fpandolfi64** [globaleaks/globaleaks-whistleblowing-software#3440] [General]
Buongiorno,
Abbiamo la necessita di leggere da una nostra applicazione le risposte di un determinato questionario ma abbiamo verificato che non è possibile farlo utilizzando il DB poichè giustamente queste sono memorizzate con crittografia.  Potete indicarci come procedere in merito? Esistono delle API da richiamare e in caso positivo potreste indicarci la modalità di attivazione?
Nel caso in cui non sia possibile farlo tramite chiamate ad API potete fornirci l'algoritmo di decript?

grazie
Fabrizio Pandolfi

**Comment by evilaliv3** [comment]
@fpandolfi64 : l'intero software funziona tramite API Rest che sono utilizzate dal client ufficiale che al momento sono poco documentate in quanto solo usate dal client ufficiale.

Per usare dovreste creare una utenza dedicate, usare le API di autenticazione che automaticamente sploccano le chiavi a sistema che sono usate poi dalle API di accesso ai dati per decifrarli automaticamente.  E' qualcosa però per il momento probabilmente molto oneroso per voi in termini di sviluppo e mainatince.

Prevediamo di estendere e standardizzare l'API per un uso come quello che descrivete ma ci impieghera ancora tempo.
Se poteste condividere maggiormente le vostre necessità questo ci aiuta a comprendere le necessità utente ed assecondarle nei nostri sviluppi. Grazie
